tracer/src/Datadog.Trace/ClrProfiler/IDistributedTracer.cs

```diff
@@ -19,9 +19,9 @@ internal interface IDistributedTracer
 
         void SetSpanContext(IReadOnlyDictionary<string, string> value);
 
-        SamplingPriority? GetSamplingPriority();
+        int? GetSamplingPriority();
```

Comment:
Is it safe to make this change? I'm not sure which of these interfaces is safe to make changes to

Reply:
I think @kevingosse said I can change `IDistributedTracer` and we should never change `ICommonTracer` or `IAutomaticTracer`. (We should add comments to those.)

Reply:
Ok that sounds about right to me, since the IDistributedTracer is only used on each side and it's the `IAutomaticTracer` and `ICommonTracer` that get duck cast on the opposite sides

Reply:
Resolving this comment. Thanks for being thorough!